cli.js

```diff
@@ -114,7 +114,19 @@ ee.on('finish', async () => {
 	ora.text = 'Code signing DMG';
 
 	try {
-		await execa('codesign', ['--sign', 'Developer ID Application', dmgPath]);
+		let identity;
+		const {stdout} = await execa('security', ['find-identity', '-v', '-p', 'codesigning']);
+		if (stdout.includes('Developer ID Application:')) {
+			identity = 'Developer ID Application';
+		} else if (stdout.includes('Mac Developer:')) {
+			identity = 'Mac Developer';
+		} else {
+			const err = new Error();
+			err.stderr = 'No usable identity found';
+			throw err;
+		}
```

Comment:
Actually, it should not throw if no identity is found, as we support running it with no identity. This `else` can be removed.

From the readme:

> It will try to code sign the DMG, but the DMG is still created and fine even if the code signing fails, for example if you don't have a developer certificate.

Reply:
You're right. I was confused.